nixos/tests/authelia.nix

```diff
@@ -0,0 +1,172 @@
+# Test Traefik as a reverse proxy of a local web service
+# and a Docker container.
+import ./make-test-python.nix ({ pkgs, ... }: {
+  name = "authelia";
+  meta = with pkgs.lib.maintainers; {
+    maintainers = [ jk ];
+  };
```

Comment:
```suggestion
import ./make-test-python.nix ({ lib, pkgs, ... }: {
  name = "authelia";
  meta.maintainers = [ pkgs.lib.maintainers.jk ];
};
```

Reply:
Please use `meta.maintainers = with lib.maintainers; [ jk ];` after importing lib directly. Using `with` in such a limited scope is not bad per se, and allows more maintainers to easily add/remove themselves without carrying the scoping around.

Reply:
Is there anyone else interested in maintaining this module? O.O

Imho the maintainers of the package should automatically be maintainers of modules, but there are some practical problems.